<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_traces_edit</class>
 <widget class="QDialog" name="Dialog_traces_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>234</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>234</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>234</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Trace list</string>
  </property>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new trace</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>49</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QGroupBox" name="groupBoxRoute">
   <property name="geometry">
    <rect>
     <x>389</x>
     <y>46</y>
     <width>141</width>
     <height>174</height>
    </rect>
   </property>
   <property name="title">
    <string>Trace #</string>
   </property>
   <widget class="QPushButton" name="pushButton_editOrLock">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>37</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit / Lock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_remove">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>135</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>77</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rename</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_showAlone">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>11</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show alone</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton_editOrLock</tabstop>
  <tabstop>pushButton_rename</tabstop>
  <tabstop>pushButton_remove</tabstop>
  <tabstop>checkBox_showAlone</tabstop>
  <tabstop>pushButton_add</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
